backend/controller/controller.go

```diff
@@ -821,6 +823,18 @@ func (s *Service) AcquireLease(ctx context.Context, stream *connect.BidiStream[f
 }
 
 func (s *Service) Call(ctx context.Context, req *connect.Request[ftlv1.CallRequest]) (*connect.Response[ftlv1.CallResponse], error) {
+	logger := log.FromContext(ctx)
+
+	requestCounter, err := otel.GetMeterProvider().Meter("ftl_call").Int64Counter(
```

Comment:
do we want this meter to be `ftl.call` to match the spreadsheet names?

Reply:
yes :)